runtime/pangolin/src/pallets/governance/track.rs

```diff
@@ -0,0 +1,236 @@
+// This file is part of Darwinia.
+//
+// Copyright (C) 2018-2023 Darwinia Network
+// SPDX-License-Identifier: GPL-3.0
+//
+// Darwinia is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Darwinia is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
+
+// core
+use core::str::FromStr;
+// darwinia
+use super::*;
+
+// TODO: move to common-runtime
+const fn percent(x: i32) -> sp_runtime::FixedI64 {
+	sp_runtime::FixedI64::from_rational(x as u128, 100)
+}
+const fn permill(x: i32) -> sp_runtime::FixedI64 {
+	sp_runtime::FixedI64::from_rational(x as u128, 1000)
+}
+
+const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 5] = [
+	(
+		0,
+		pallet_referenda::TrackInfo {
+			// Name of this track.
+			name: "root",
+			// A limit for the number of referenda on this track that can be being decided at once.
+			// For Root origin this should generally be just one.
+			max_deciding: 5,
+			// Amount that must be placed on deposit before a decision can be made.
+			decision_deposit: DARWINIA_PROPOSAL_REQUIREMENT,
+			// Amount of time this must be submitted for before a decision can be made.
+			prepare_period: 2 * MINUTES,
+			// Amount of time that a decision may take to be approved prior to cancellation.
+			decision_period: 2 * MINUTES,
```

Comment:
I see that you have shorten lots of time config from 10 mins to 2 mins. After I did some local tests, the feedback is that 2 mins is too short, you can not even understand what happened, the time is out and proposal has been rejected.  So I suggest making the time value larger.